packages/server/src/user/user.service.ts

```diff
@@ -13,17 +13,17 @@ export class UserService {
   /**
    * Register a new record of user in the database
    *
-   * @param data object that should contain `projectId: string`, `username?: string`, `email?: string` and `password: string` in plain text.
+   * @param newUser object that should contain `projectId: string`, `username?: string`, `email?: string` and `password: string` in plain text.
    * **NOTE:** `data` should contain either `username`, `email`, or both.
```

Comment:
NIT: Changing `data` to `newUser`.

Reply:
Just saw the `data` in the comment hasn't been changed. It's fixed now.